### PR TITLE
Update processing rules

### PR DESCRIPTION
**Pull request by PieterKas**
See #49

### CODE REVIEW

File: identity-chaining-draft.md [hunk 1]
```diff
@@ -254,9 +254,9 @@ The client MAY indicate the audience it is trying to access through the `scope`
 
 ### Processing rules
 
-All of {{RFC7521}} (Section 5.2 in specific) applies, along with the following processing rules:
+All of Section 5.2 {{RFC7521}} applies, in addition to the following processing rules:
 
-* The request MUST be denied if the presented authorization grant does not include an "aud" claim identifying the authorization server that processes the request.
+* The "aud" claim value MUST be either the token endpoint identifier as described Section 3 {{RFC7523}} or the Authorization Server issuer idetifier as defined in Section 5.2 of {{RFC7521}}.
```
Comment (bc-pi) [inline identity-chaining-draft.md:259]: idetifier -> identifier  

but Section 5.2 of RFC7521 doesn't define the issuer identifier, does it? I think "issuer identifier" is defined first in AS Metadata RFC8414

Comment (bc-pi) [reply identity-chaining-draft.md:259]: https://datatracker.ietf.org/doc/html/rfc9126#section-2-5 or https://www.rfc-editor.org/rfc/rfc9207#section-1.1-2 are examples of places that cite it

Comment (PieterKas) [reply identity-chaining-draft.md:259]: Aha! That makes a lot more sense, thanks Brian. Let me try again.

Comment (PieterKas) [reply identity-chaining-draft.md]: @bc-pi I made updates to try and clarify that:

1. The aud claim should identify the token endpoint or the authorization server
2. The issuer identifier should follow the format defined in RFC 8414

Thanks for pointing that out, I would never have known to look there otherwise.

Please suggest alternative text in case the change is not clear.

Comment (bc-pi) [reply identity-chaining-draft.md]: Thinking about it a bit more and this is trickier than it should be- sorry! But I think the language you had before is preferable with just getting the reference and spelling right. I'll suggest some text

File: identity-chaining-draft.md [hunk 2]
```diff
@@ -254,9 +255,9 @@ The client MAY indicate the audience it is trying to access through the `scope`
 
 ### Processing rules
 
-All of {{RFC7521}} (Section 5.2 in specific) applies, along with the following processing rules:
+All of Section 5.2 {{RFC7521}} applies, in addition to the following processing rules:
 
-* The request MUST be denied if the presented authorization grant does not include an "aud" claim identifying the authorization server that processes the request.
+* The "aud" claim MUST identify either the token endpoint or the authorization server. To avoid ambiguity, the issuer identifier as defined in {{RFC8414}} SHOULD be used as the value of the "aud" claim.
```
Comment (bc-pi) [inline identity-chaining-draft.md:260]:
```suggestion
* The "aud" claim value MUST identify the Authorization Server as a valid intended audience of the assertion using either the token endpoint as described Section 3 {{RFC7523}} or the issuer identifier as defined in Section 2 of {{RFC8414}}.
```

File: identity-chaining-draft.md (hunk 1 above)
Comment (bc-pi) [reply identity-chaining-draft.md]: I'm even kinda having second thoughts about the value/appropriateness of bringing up RFC 8414 here but it's probably okay...

**Review by arndt-s — APPROVED** [review]
